plasmapy/formulary/radiation.py

```diff
@@ -0,0 +1,159 @@
+"""
+Defines the Radiation module as part of the Plasma Formulary
+"""
+
+__all__ = [
+    "thermal_bremsstrahlung",
+]
+
+import astropy.constants as const
+import astropy.units as u
+import numpy as np
+
+from scipy.special import exp1
+from typing import List, Tuple, Union
+
+from plasmapy.formulary.parameters import plasma_frequency
+from plasmapy.particles import Particle
+from plasmapy.utils.decorators import validate_quantities
+from plasmapy.utils.exceptions import PhysicsError
+
+
+@validate_quantities(
+    frequencies={"can_be_negative": False},
+    ne={"can_be_negative": False},
+    ni={"can_be_negative": False},
+    Te={"can_be_negative": False, "equivalencies": u.temperature_energy()},
+)
+def thermal_bremsstrahlung(
+    frequencies: u.Hz,
+    ne: u.m ** -3,
+    Te: u.K,
+    ni: u.m ** -3 = None,
+    ion_species: Union[str, Particle] = "H+",
```

Comment:
Here we can use the `@particle_input` decorator from `plasmapy.particles`, which will automatically convert a string representation of a particle into a `Particle` object.  If I remember correctly, the decorator will make sure that the particle is an ion if the argument is named `ion`.
```suggestion
    ion: Particle = None,
```

I've also been thinking that we should avoid setting default particles, since lab plasma physicists may assume a different default particle than space plasma physicists.

Reply:
Good idea: I'll use the decorator. I see your point about the potential for confusion, but as a lab plasma physicist I think H+ would still be my assumption? The benefit of having a default is the ability to do a quick calculation without having to be familiar with the particle class.